WordPress/Classes/Users/Views/UserDetailsView.swift

```diff
@@ -113,12 +121,8 @@ struct UserDetailsView: View {
         .deleteUser(in: self)
         .onAppear() {
             Task {
-                await viewModel.loadCurrentUserRole()
+                await viewModel.onAppear()
                 await deleteUserViewModel.fetchOtherUsers()
```

Comment:
Does `await deleteUserViewModel.fetchOtherUsers()` need to happen here?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23834/commits/af1180de6331591410e809c6527f6c2adccca874.